docs/source/components/nodes/object_tracker.rst

```diff
@@ -1,7 +1,7 @@
 ObjectTracker
 =============
 
-Object tracker tracks detected objects from the :ref:`ImgDetections` using Kalman filter and hungarian algorithm.
+Object tracker tracks detected objects from the :ref:`ImgDetections`.
```

Comment:
What's the purpose of removing that?
KCF is experimental, and not real-time, so the main one suitable for real time tracking is with kalman filter

Reply:
Maybe we should make it more explicit .